admin-auth/src/main/kotlin/com/epam/drill/admin/auth/service/impl/CacheServiceImpl.kt

```diff
@@ -0,0 +1,26 @@
+package com.epam.drill.admin.auth.service.impl
+
+import com.epam.drill.admin.auth.service.CacheService
+import com.github.benmanes.caffeine.cache.Cache
+import io.ktor.auth.*
+
+/**
+ * Implementation of [CacheService], providing caching functionalities.
+ *
+ * @property cache An instance of Caffeine`s [Cache] that stores key-value pairs, where keys are of type [String] and values are [Principal] objects.
+ */
+class CacheServiceImpl(
```

Comment:
In theory, we can have more than one implementation of a caching service. And it would be more logical to use their cache provider in the name. For this case it would be `CoffeineCacheService`

Reply:
renamed